<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>command_window</class>
 <widget class="QWidget" name="command_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>144</width>
     <height>193</height>
    </rect>
   </property>
   <property name="title">
    <string>传感器控制</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>EN_DVS</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="EN_DVS_CB">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>EN_APS</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="EN_APS_CB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>DVS_GAIN</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="DVS_GAIN_CB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>DI_CAP_SW</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="DI_CAP_SW_CB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>VGPOL_SW</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="VGPOL_SW_CB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>DVS_TOP_RSTN</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="DVS_TOP_RSTN_CB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="cmd_set_pB">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>291</width>
     <height>701</height>
    </rect>
   </property>
   <property name="title">
    <string>FA2415A SPI</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>244</width>
      <height>668</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="0" column="0" rowspan="28">
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2" colspan="4">
      <widget class="Line" name="line_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2" colspan="2">
      <widget class="Line" name="line_11">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>OUT1&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>ROW_START&lt;0:6&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>OPEN_WIN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>LIGHT_CURRENT</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>COL_START&lt;0:6&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>OUT2&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="2" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>SEL&lt;0:1&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>ADJ_TH</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>OUT3&lt;0:5&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>EN DVS</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>EN APS</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>SPI DVS RONGHE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="2" colspan="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="2" rowspan="2" colspan="2">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>OUT4&lt;3:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>CHOUZHEN START&lt;0:3&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="2" colspan="3">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="2" colspan="3">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>OUT5&lt;3 :6&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>CHOUZHEN START&lt;4:7&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="16" column="2">
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="1" column="1">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>IN CTRL&lt;0:3&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>OUT6&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>IP CTRL&lt;0:3&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="17" column="2">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="3" column="1">
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>RN</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>LPF EN</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>OUT7&lt;0 :6&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>PFD EN</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>MUX&lt;0 : 3&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="18" column="2" colspan="3">
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>OUT8&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>OUT11&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>OUT10&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>OUT9&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="2">
        <widget class="QLabel" name="label_38">
         <property name="text">
          <string>SEL C1&lt;1:32&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="21" column="2" colspan="3">
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="22" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>OUT13&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>OUT12&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>SEL C2&lt;1:16&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="24" column="2" colspan="3">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="25" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="1" rowspan="2">
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>VCO PVT&lt;1:14&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>OUT15&lt;0:5&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>OUT14&lt;0:7&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="27" column="2" colspan="3">
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>181</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>dvs64_spiconfg</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_18">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>on</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_42">
           <property name="text">
            <string>去噪算法</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="on_Denos_on_rB">
           <property name="text">
            <string>开</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="on_Denos_off_rB">
           <property name="text">
            <string>关</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_43">
           <property name="text">
            <string>滑窗值</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="on_SlidWD_3_rB">
           <property name="text">
            <string>3</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="on_SlidWD_5_rB">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_44">
           <property name="text">
            <string>去噪阈值</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QLineEdit" name="on_DenosTrsd_lE">
           <property name="text">
            <string>16</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QPushButton" name="on_set_pB">
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>off</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_17">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_45">
           <property name="text">
            <string>去噪算法</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="off_Denos_on_rB">
           <property name="text">
            <string>开</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="off_Denos_off_rB">
           <property name="text">
            <string>关</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_46">
           <property name="text">
            <string>滑窗值</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="off_SlidWD_3_rB">
           <property name="text">
            <string>3</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="off_SlidWD_5_rB">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_47">
           <property name="text">
            <string>去噪阈值</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QLineEdit" name="off_DenosTrsd_lE">
           <property name="text">
            <string>16</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QPushButton" name="off_set_pB">
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
